### PR TITLE
Fix bug for version update hint

### LINKED ISSUE #18698: Error 500 on admin users dashboard when there is a new version
### Gitea Version

itea version 1.16.0+rc1 built with GNU Make 4.3, go1.17.6 : bindata, timetzdata, sqlite, sqlite_unlock_notify

### Git Version

git version 2.30.2

### Operating System

Docker（QNAP Container Station）

### How are you running Gitea?

 User Docker（QNAP Container Station）。Reference https://www.anchorpoint.app/blog/setting-up-a-self-hosted-git-server

### Database

PostgreSQL

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

_No response_

### Description

I've been using it without problems until today。
When I access the **site administration** page, I get a 500 error 。

log：
```
2022/02/10 10:32:20 Completed GET /user/events 200 OK in 14m13.091847516s
2022/02/10 10:32:20 ...s/context/context.go:206:HTML() [E] Render failed: template: admin/dashboard:8:90: executing "admin/dashboard" at <Str2html>: invalid value; expected string
2022/02/10 10:32:20 Completed GET /admin 500 Internal Server Error in 42.299935ms
2022/02/10 10:32:21 Started GET /user/events for 192.168.5.51:55427
```

### Screenshots

image

### PR DESCRIPTION
**Pull request by lunny**
Fix #18698

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -2382,6 +2382,7 @@ first_page = First
 last_page = Last
 total = Total: %d
 
+dashboard.new_version_hint = Gitea %s is now available, you are running %s. Check the <a href="https://blog.gitea.io">blog</a> for more details.
```
Comment (silverwind) [inline options/locale/locale_en-US.ini]:
```suggestion
dashboard.new_version_hint = Gitea %s is now available, you are running %s. Check the <a target="_blank" rel="noreferrer" href="https://blog.gitea.io">blog</a> for more details.
```

Comment (wxiaoguang) [reply options/locale/locale_en-US.ini]: `target="_blank"` implies `noopener`, it can be simplified.

Comment (silverwind) [reply options/locale/locale_en-US.ini]: I'm not sure that's true but we have a lot of existing similar links that set `noopener` in localization.

Comment (wxiaoguang) [reply options/locale/locale_en-US.ini]: https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

image

Comment (silverwind) [reply options/locale/locale_en-US.ini]: Interesting, didn't know that. We should probably refactor translations, or better yet, just have some logic in the i18n code that automatically adds attributes to all external `<a>` elements automatically.

Comment (silverwind) [reply options/locale/locale_en-US.ini]: Edited the suggestion.

**Review by zeripath — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Pull request merged**